### PR TITLE
4.18

### PR DESCRIPTION
**Pull request by DaanHoogland**

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
@blueorangutan docbuild

File: source/_global.rst [hunk 1]
```diff
@@ -17,27 +17,27 @@
 .. This file contain Variables shared across other .rst files in this project
 
 
-.. _CloudStack Installation Guide: https://docs.cloudstack.apache.org/projects/cloudstack-installation
-.. _CloudStack Administrator's Guide: https://docs.cloudstack.apache.org/projects/cloudstack-administration
+.. _CloudStack Installation Guide: http://docs.cloudstack.apache.org/en/latest/installguide/index.html
+.. _CloudStack Administrator's Guide: http://docs.cloudstack.apache.org/en/latest/adminguide/index.html
```
Comment (rohityadavcloud) [inline source/_global.rst:21]: should this be https?

Comment (rohityadavcloud) [inline source/_global.rst:21]:
```suggestion
.. _CloudStack Installation Guide: https://docs.cloudstack.apache.org/en/latest/installguide/index.html
.. _CloudStack Administrator's Guide: https://docs.cloudstack.apache.org/en/latest/adminguide/index.html
```

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

**Pull request merged**